### PR TITLE
Add myself to FUNDING.yml and fix maisy's username

### PR DESCRIPTION
**Pull request by wingio**
shiggy

### CODE REVIEW

**Review by Cynosphere — APPROVED** [review]

File: FUNDING.yml [hunk 1]
```diff
@@ -1 +1 @@
-github: [Beefers]
+github: [maisymoe, wingio]
```
Comment (zt64) [inline FUNDING.yml:1]:
```suggestion
github: [maisymoe, wingio, zt64]
```

**Review by zt64 — CHANGES_REQUESTED** [review]

File: FUNDING.yml (hunk 1 above)
Comment (wingio) [reply FUNDING.yml:1]: nop

Comment (Vendicated) [inline FUNDING.yml:1]:
```suggestion
github: [maisymoe, wingio, mwittrien]
```

Comment (zt64) [reply FUNDING.yml:1]: nop

**Pull request merged**